document-store/src/main/java/org/hypertrace/core/documentstore/model/options/UpdateOptions.java

```diff
@@ -9,7 +9,16 @@
 @Builder
 public class UpdateOptions {
   public static UpdateOptions DEFAULT_UPDATE_OPTIONS =
-      UpdateOptions.builder().returnDocumentType(AFTER_UPDATE).build();
+      UpdateOptions.builder()
+          .returnDocumentType(AFTER_UPDATE)
+          .missingDocumentStrategy(MissingDocumentStrategy.SKIP_UPDATES)
+          .build();
 
   ReturnDocumentType returnDocumentType;
```

Comment:
```suggestion
  @Default ReturnDocumentType returnDocumentType = AFTER_UPDATE;
```

Reply:
Since we are having multiple properties now, it's better to define defaults for each one individually.